File: app/src/main/java/ca/georgian/assignment2/MyViewHolder.java
package ca.georgian.assignment2;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import  androidx.recyclerview.widget.RecyclerView;

public class MyViewHolder extends RecyclerView.ViewHolder{

    TextView title;
    TextView studio;
    TextView rating;
    TextView year;

    public MyViewHolder(@NonNull View itemView) {
        super(itemView);
        title = itemView.findViewById(R.id.movieTitle);
        studio = itemView.findViewById(R.id.movieStudio);
        rating = itemView.findViewById(R.id.movieRating);
        year = itemView.findViewById(R.id.movieYear);
    }
}

File: app/src/main/java/ca/georgian/assignment2/MyViewHolder.kt
package ca.georgian.assignment2

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView

class MyViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    val title: TextView = itemView.findViewById(R.id.movieTitle)
    val studio: TextView = itemView.findViewById(R.id.movieStudio)
    val rating: TextView = itemView.findViewById(R.id.movieRating)
    val year: TextView = itemView.findViewById(R.id.movieYear)
    val poster: ImageView = itemView.findViewById(R.id.moviePoster)
}